src/plugins/drawing-tool/components/drawing-toolbar.tsx

```diff
@@ -22,7 +22,7 @@ interface IProps extends IFloatingToolbarProps, IRegisterToolApiProps {
   setImageUrlToAdd: (url: string) => void;
 }
 
-const defaultButtons = ["select", "line", "vector", "rectangle", "ellipse", 
```

Comment:
I have `enable automatic trim trailing whitespace` on my VSCode which is why these were added.

Reply:
That's fine -- so do I.